datadog.go

```diff
@@ -0,0 +1,137 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// Package datadog contains a Datadog exporter.
+package datadog
+
+import (
+	"log"
+	"regexp"
+	"strings"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+const (
+	defaultHost   = "localhost"
+	defaultPort   = "8125"
+	opencensusTag = "source:Opencensus"
+)
+
+var (
+	exporter *Exporter
+	tags     = []string{opencensusTag}
+	reg      = regexp.MustCompile("[^a-zA-Z0-9]+")
+)
+
+// Exporter exports stats to Datadog.
+type Exporter struct {
+	opts      Options
```

Comment:
This field is not needed because it's not used anywhere. It can be removed. It's enough to pass it to `statsExporter` (or `collector` if you decide to keep that name, which is perfectly fine) as you already did.